### PR TITLE
UF-5103: Extended messaging request to include needed information

### PR DESCRIPTION
**Pull request by josc04**
- Added mandatory attachment deliverymode attribute
- Added mandatory body content attribute
- Added mandatory body contenttype attribute
- Added optional subject information attribute
- Added optional support information attribute

### CODE REVIEW

File: src/main/java/se/sundsvall/processengine/parkingpermit/integration/messaging/mapper/MessagingMapper.java [hunk 1]
```diff
@@ -1,21 +1,36 @@
...
 import generated.se.sundsvall.messaging.WebMessageRequest;
 import generated.se.sundsvall.templating.RenderResponse;
 
 public class MessagingMapper {
+	private static final Encoder ENCODER = Base64.getEncoder();
+
 	private static final String DEPARTMENT = "SBK(Gatuavdelningen, Trafiksektionen)";
 	private static final String FILE_NAME = "beslut.pdf";
 	private static final String MESSAGE = "Ärendet avskrivs";
+	private static final String SUBJECT = "Beslut från Sundsvalls kommun";
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/processengine/parkingpermit/integration/messaging/mapper/MessagingMapper.java:28]: Should this info be externalized (e.g. in application.properties)?

(No big deal, just a thought)

File: src/main/java/se/sundsvall/processengine/parkingpermit/integration/messaging/mapper/MessagingMapper.java [hunk 2]
```diff
@@ -1,21 +1,36 @@
...
+import generated.se.sundsvall.messaging.LetterRequest.ContentTypeEnum;
+import generated.se.sundsvall.messaging.LetterSender;
+import generated.se.sundsvall.messaging.LetterSenderSupportInfo;
 import generated.se.sundsvall.messaging.WebMessageAttachment;
 import generated.se.sundsvall.messaging.WebMessageParty;
 import generated.se.sundsvall.messaging.WebMessageRequest;
 import generated.se.sundsvall.templating.RenderResponse;
 
 public class MessagingMapper {
+	private static final Encoder ENCODER = Base64.getEncoder();
```
Comment (foobar-1337) [inline src/main/java/se/sundsvall/processengine/parkingpermit/integration/messaging/mapper/MessagingMapper.java:23]: `Base64.getEncoder()` already returns a `static final Encoder`. No performance is gained here (just an extra row).

**Review by foobar-1337 — APPROVED** [review]

**Comment by foobar-1337** [comment]
Approved with some comments. Do whatever you want with them...

**Comment by josc04** [comment]
Acted on PR comments

**Review by foobar-1337 — APPROVED** [review]

**Pull request merged**